tutorials/plot_visualize_evoked.py

```diff
@@ -172,6 +172,14 @@
 mne.viz.plot_evoked_topo(evoked, title=title % 'Left/Right Auditory/Visual',
                          background_color='w')
 
+###############################################################################
+# For small numbers of sensors, it is also possible to create a more refined
+# topoplot. Again, clicking on a sensor opens a single-sensor plot.
+
+mne.viz.plot_compare_evokeds(evoked_dict, picks="eeg", colors=colors,
+                             linestyles=linestyles, split_legend=True,
+                             axes="topo")
+
```

Comment:
@agramfort what do you think of the example?

I will link sphinx when it is rendered.

If you want a new parameter instead of axes="topo", I can do that.

Reply:
here's the link
https://12707-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/plot_visualize_evoked.html